typings/reference-data-reader/index.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'reference-data-reader' {
+  export interface RoundData {
+    roundId: BigNumber
+    answer: BigNumber
+    startedAt: BigNumber
+    updatedAt: BigNumber
+    answeredInRound: BigNumber
+  }
+}
```

Comment:
We should declare this in `helpers/reference-data-reader/src/index.ts`. Declaration here is only useful when we're dealing with packages that aren't in Typescript, but in this case we can declare it directly in the source.

Reply:
moved to src